1.illumination_correction/README.md

```diff
@@ -0,0 +1,116 @@
+I chose this image because in the [CellProfiler illumination correction tutorial](https://cellprofiler-examples.s3.amazonaws.com/ExampleIlluminationCorrection_Tutorial.pdf), it gives an example of an image with a large artifact that was removed using illumination correction. 
+
+Based on this, I wanted to see how these pipelines did with an one image from the NF1 data set that contained a large artifact and assess if the pipelines could remove it like in the example.
+
+## Opinions for each pipeline
+
+Below I provide my opinions regarding each pipeline, how the illumination correction function performed on the chosen image, and how it performed in general with all of the images.
+
+### Formatting of the pipeline titles
+
+The pipelines have titles that are meant for you to input into CellProfiler in order. Here is the parameters that I changed for these pipelines and the options within them:
+
+#### CorrectIlluminationCalculate Module
+
+- "Select how the illumination function is calculated": `Regular` or `Divide`
+- "Dilate objects in the final averaged image?": `Yes` or `No`
+    - If `Yes`, set "Dilation Radius" to specific value
+- "Calculate function for each image individually, or based on all images?": `Each`, `All: Across Cycles`, or `All: First Cycle` 
+- "Smoothing Method": `No smoothing`, `Convex Hull`, `Fit Polynomial`, `Median Filter`, `Gaussian Filter`, `Smooth to Average`, or `Spline`
+- "Method to calculate smoothing filter size": `Automatic`, `Object Size`, or `Manually`
+    - If `Object Size` or `Manually`, set value for `Approximate object diameter` or `Smoothing filter size` respectively
+
+#### CorrectIlluminationApply Module
+
+- "Select how the illumination function is applied": `Divide` or `Subtract`
+
+**For more information on these modules and what the parameters do, please reference the CellProfiler documentation for the [CorrectIlluminationCalculate module](http://d1zymp9ayga15t.cloudfront.net/content/Documentation/cp2.1_2.2manual/CorrectIlluminationCalculate.html) and [CorrectIlluminationApply module](http://d1zymp9ayga15t.cloudfront.net/content/Documentation/cp2.1_2.2manual/CorrectIlluminationApply.html).**
+
+### Regular -> Each image -> Median -> Manual 40 -> Divide
+
+- Fast runtime.
+- All images are very dim, hard to tell if illumination correction occurred.
+- Must brighten images to see the objects.
+- For the chosen image, once brightened in ImageJ, it looks like the nuclei have become darker in the center and then brighter on the edges.
+    - Due to this, this pipeline is likely not viable as it could impact the biological anaylsis.
+
+### Regular -> All: Across Cycles -> Median -> Manual 40 -> Divide 
+
+- Much slower runtime, approximately 12 minutes.
+- Creates a weird "shadow" effect, where in all the images, there looks to be dark shadows of the object being stained for based on the channel. 
+See example below from the DAPI channel:
+
+![Shadow Example](Examples/Shadow_Example.png)
```

Comment:
This is really nice to see this example image! Why not have example images for all? (i know it's a bunch more work, so if you don't think having examples is necessary, please don't add them)

Reply:
I am going to hold off on the adding example images for all of them. I picked those two because the images that they have from the "chosen images" did not show the shadow or faint box issue very well so I needed to find better examples. For the other pipelines, the images they have available will show the issues well.